docs/overview/status/data.json

```diff
@@ -310,6 +310,12 @@
       "status": "✅",
       "notes": ""
     },
+    {
+      "name": "Facet Search",
+      "features": "",
+      "status": "Supported",
+      "notes": "See the Facet Search Sample"
```

Comment:
I think we should just specify that the facet search sample in the accordion section: "See the Facet Search Sample located in the Accordion section" (or something similar).